Shared/SharedUI/BaseDialogs/StandardDialog/PackFile/PackFileBrowserWindow.xaml.cs

```diff
@@ -48,9 +46,19 @@ private void ViewModel_FileOpen(PackFile file)
             Close();
         }
 
+        private void ViewModel_NodeSelected(TreeNode node)
```

Comment:
What is the purpose of this? how does this work?

Reply:
It might make more sense if it was seen in the context of the AudioEditor, but it's to have a way of capturing the selected folder as the SelectedNode isn't always a folder. Though come to think of it, I suppose given that a folder is not a possible result in any of the other pack file browsing setups and when using the folder only browsing option only folders are a possible result then maybe SelectedNode could be used directly?